src/main/pg/bus_spi.c

```diff
@@ -101,11 +101,8 @@ ioTag_t preinitIPUList[SPI_PREINIT_IPU_COUNT] = {
 #ifdef SDCARD_SPI_CS_PIN
     IO_TAG(SDCARD_SPI_CS_PIN),
 #endif
-#ifdef BMP280_CS_PIN
-    IO_TAG(BMP280_CS_PIN),
-#endif
-#ifdef MS5611_CS_PIN
-    IO_TAG(MS5611_CS_PIN),
+#ifdef BARO_CS_PIN
+    IO_TAG(BARO_CS_PIN),
```

Comment:
This will not work for universal targets, we need to get the pin from `barometerConfig->baro_spi_csn`.